include/sos/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
this file will not be able to generate a Makefile in other computers than yours, please remove explicit paths